src/main/java/com/github/greengerong/PreRenderSEOFilter.java

```diff
@@ -14,7 +14,7 @@
 public class PreRenderSEOFilter implements Filter {
     public static final List<String> PARAMETER_NAMES = Lists.newArrayList("preRenderEventHandler", "proxy", "proxyPort",
             "prerenderToken", "forwardedURLHeader", "crawlerUserAgents", "extensionsToIgnore", "whitelist",
-            "blacklist", "prerenderServiceUrl");
+            "blacklist", "prerenderServiceUrl","socketTimeout");
```

Comment:
Since socketTimeout was not a 'whitelisted' parameter, override from within our filters did not work. Adding this enabled us to specify socket timeouts for callouts to prerender.